gpc-api-facade/src/integrationTest/java/uk/nhs/adaptors/pss/gpc/controller/PatientTransferControllerIT.java

```diff
@@ -0,0 +1,120 @@
+package uk.nhs.adaptors.pss.gpc.controller;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.boot.test.context.SpringBootTest.WebEnvironment.RANDOM_PORT;
+import static org.springframework.http.HttpStatus.ACCEPTED;
+import static org.springframework.http.HttpStatus.NO_CONTENT;
+
+import static io.restassured.RestAssured.given;
+import static uk.nhs.adaptors.pss.gpc.utils.TestResourceUtils.readResourceAsString;
+
+import javax.jms.Message;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.hl7.fhir.dstu3.model.Identifier;
+import org.hl7.fhir.dstu3.model.Parameters;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import lombok.SneakyThrows;
+import uk.nhs.adaptors.connector.dao.PatientMigrationRequestDao;
+import uk.nhs.adaptors.connector.model.PatientMigrationRequest;
+import uk.nhs.adaptors.connector.model.RequestStatus;
+import uk.nhs.adaptors.pss.gpc.containers.IntegrationTestsExtension;
+import uk.nhs.adaptors.pss.gpc.service.FhirParser;
+import uk.nhs.adaptors.pss.gpc.utils.QueueUtils;
+
+@SpringBootTest(webEnvironment = RANDOM_PORT)
+@ExtendWith({SpringExtension.class, IntegrationTestsExtension.class})
+@DirtiesContext
+public class PatientTransferControllerIT {
+    private static final String APPLICATION_FHIR_JSON_VALUE = "application/fhir+json";
+    private static final String MIGRATE_PATIENT_RECORD_ENDPOINT = "/Patient/$gpc.migratestructuredrecord";
+    private static final String VALID_REQUEST_BODY_PATH = "/requests/migrate-patient-record/validRequestBody.json";
+
+    @LocalServerPort
+    private int port;
+
+    @Autowired
+    private PatientMigrationRequestDao patientMigrationRequestDao;
+
+    @Autowired
+    private QueueUtils queueUtils;
+
+    @Autowired
+    private FhirParser fhirParser;
+
+    @Test
+    public void migratePatientStructuredRecord() {
```

Comment:
Maybe a more meaningful name for the scenario? Or consider splitting into two cases?